app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,43 @@
+class CommentsController < ApplicationController
+  before_action :logged_in_user, only: [:create, :destroy]
+  before_action :correct_user, only: [:create, :destroy]
+  def show
+    @recruiting = Recruiting.find(params[:id])
+    @comments = Comment.where(recruiting_id: params[:id])
+  end
+
+  def create
+    @reply = Comment.new(reply_params)
+    if @reply.save
+      flash[:success] = "返信しました"
+      redirect_to comment_path(reply_params[:recruiting_id])
+    else
+      id = reply_params[:recruiting_id]
+      @recruiting = Recruiting.find(id)
+      @comments = Comment.where(recruiting_id: id)
+      @error_message = "この内容では返信できません"
+      render 'show'
+    end
+  end
+
+  def destroy
+    comment = Comment.find(params[:id])
+    recruiting = Recruiting.find_by(id: comment.recruiting_id)
+    respond_to do |format|
+      if comment.destroy
+        flash[:success] = "返信を削除しました"
+        format.html { redirect_to comment_path(recruiting.id) }
```

Comment:
`recruiting = Recruiting.find_by(id: comment.recruiting_id)` 消して下記でいいです。

```suggestion
        format.html { redirect_to comment_path(comment.recruiting_id) }
```